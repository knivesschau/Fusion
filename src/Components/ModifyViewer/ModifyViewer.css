#modify-info { 
    font-size: x-large;
}

.Title_Modifier, .Ingredient_Modifier, .Steps_Modifier {
    border: 1px solid black;
    padding: .5em;
    margin-bottom: 1em;
}

#fuse_name_mod {
    width: 90%;
}

#mod_ingredients {
    margin: .25em;
    width: 90%;
}

#mod-steps-list {
    padding-left: 1em;
}

#mod_steps {
    vertical-align: middle;
    width: 80%;
    margin: .25em;
}