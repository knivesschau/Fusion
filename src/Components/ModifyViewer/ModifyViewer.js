import React, { Component } from 'react';
import fusionContext from '../../fusionContext';
import './ModifyViewer.css';

export default class ModifyViewr extends Component {
    render() {
        return (
            <>
            </>
        );
    };
}