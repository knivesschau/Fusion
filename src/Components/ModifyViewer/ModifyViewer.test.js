import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import ModifyViewer from './ModifyViewer';

describe ('ModifyViewer Component', () => {
    it ('Renders the complete ModifyViewer Component and all nested Components', () => {
        const wrapper = shallow(<ModifyViewer/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});