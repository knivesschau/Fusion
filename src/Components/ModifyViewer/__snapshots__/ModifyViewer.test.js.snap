// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModifyViewer Component Renders the complete ModifyViewer Component and all nested Components 1`] = `<Fragment />`;
